mmseg/models/backbones/hrnet.py

```diff
@@ -575,6 +607,7 @@ def train(self, mode=True):
         """Convert the model into training mode will keeping the normalization
         layer freezed."""
         super(HRNet, self).train(mode)
+        self._freeze_stages()
```

Comment:
Move this line inside `train()`.

Reply:
@xvjiarui  I did not find any difference between the `resnet` and `hrnet` about `self._freeze_stages()`. Could you give more information?

![image](https://user-images.githubusercontent.com/5787131/127949004-176b9e5e-8cd7-47f9-829f-c47755b3efcf.png)

Reply:
Sorry. My bad, I forgot to expand the collapsed code lines.